### ISSUE TITLE
Document which reporting commands are not affected by context

### ISSUE THREAD

**Issue by tbabej** [GothenburgBitFactory/taskwarrior#2099]
As #2067 points out, `ids` commands is not affected by context. This is by design, and there are other commands like this. Currently this is only documented in the source code (see uses_context):

```
CmdIDs::CmdIDs ()
{
  _keyword               = "ids";
  _usage                 = "task <filter> ids";
  _description           = "Shows the IDs of matching tasks, as a range";
  _read_only             = true;
  _displays_id           = true;
  _needs_gc              = true;
  _uses_context          = false;
  _accepts_filter        = true;
  _accepts_modifications = false;
  _accepts_miscellaneous = false;
  _category              = Command::Category::metadata;
}

```

All reporting commands (commands that take filters) that are not affected by context should be explicitly marked as such in the man page.

**Comment by mrossinek** [comment]
This was fixed with #2103.
Sorry, I forgot to use a keyword in my commit and PR.

**Comment by tbabej** [comment]
Thanks for noting that Max. Closing.
